packages/core/src/Stores/client-store.js

```diff
@@ -364,10 +364,17 @@ export default class ClientStore extends BaseStore {
         return 'USD';
     }
 
+    @computed
+    get allow_authentication() {
+        return this.account_status?.status?.some(status => status === 'allow_document_upload');
+    }
+
     @computed
     get is_authentication_needed() {
-        if (!this.account_status.status) return false;
-        return this.account_status.authentication.needs_verification.length;
+        return (
+            this.account_status?.authentication?.needs_verification?.length ||
+            !!this.account_status?.prompt_client_to_authenticate
```

Comment:
i dont think we use this status anymore in the future. checking only needs_verification length should be ok